client.go

```diff
@@ -53,6 +66,40 @@ func Timeout(timeout time.Duration) func(*Client) error {
 	}
 }
 
+// Keepalive keeps the connection open.
+func Keepalive() func(*Client) error {
+	return func(c *Client) error {
+		go func(c *Client) {
+			for c.IsConnected() {
```

Comment:
Do we want to be more intelligent that this and only send if there has been no traffic.

Also is there are reason you're not just using: https://golang.org/pkg/net/#TCPConn.SetKeepAlive ?

Reply:
The TeamSpeak Server Query closes the connection after five minutes of inactivity.
The only way to keep the connection open is by executing a command (doesn't have to be valid) or by sending a space + new line.
I'm pretty sure that this means that we cannot use `TCPConn.SetKeepAlive()` as **it does not send any "real" data** and would therefore not keep the connection open.

Reply:
cool, thanks